### ISSUE TITLE
action step generate_dynamic_content fails with "TypeError: decode() takes 1 positional argument but 2 were given"

### ISSUE THREAD

**Issue by jaymccon** [aws-quickstart/quickstart-documentation-base-common#85]
```
---
  File "<BASE_PATH>/docs/boilerplate/.utils/generate_parameter_tables.py", line 9, in get_cfn
Working on templates/eks-snyk.template.yaml
    _decoded = cfnlint.decode.decode(filename, False)[0]
TypeError: decode() takes 1 positional argument but 2 were given
Error: Process completed with exit code 1.
```

I'm guessing this is due to an upstream change in `cfnlint`

**Comment by jaymccon** [comment]
verified that the error only occurs with cfnlint 0.39.0 . Previous version 0.38.0 works as expected

**Comment by jaymccon** [comment]
fixed by #87
